thetis/coordsys.py

```diff
@@ -5,17 +5,133 @@
 import firedrake as fd
 import pyproj
 import numpy
+from abc import ABC, abstractmethod
 
-
-UTM_ZONE10 = pyproj.Proj(
-    proj='utm',
-    zone=10,
-    datum='WGS84',
-    units='m',
-    errcheck=True)
 LL_WGS84 = pyproj.Proj(proj='latlong', datum='WGS84', errcheck=True)
 
 
+class MeshCoordSystem(ABC):
```

Comment:
or `MeshCoordinateSystem`, `CoordinateSystem`?

Reply:
I would prefer `CoordinateSystem` because the (sub)class doesn't depend on a mesh.

Reply:
I agree, changed.